node_modules/node-twitter-api/.npmignore

```diff
@@ -1,2 +0,0 @@
-npm-debug.log
-node_modules
```

Comment:
Since I use vim, I prefer to have an ending newline at the end of files (otherwise vim will add one next time I edit, and create a spurious diff).